docs/src/js/base/menu-button.js

```diff
@@ -20,8 +20,13 @@ function onMenuKeyDown(e) {
     }
 }
 
-function onMenuItemSelect() {
+function onMenuItemSelect(e) {
+    if (this._buttonPrefix && e.detail.el.dataset?.menuitemradioValue) {
+        this._buttonTextEl.innerText = this._buttonPrefix + e.detail.el.dataset?.menuitemradioValue;
```

Comment:
I'm glad we added optional chaining; I think it makes things easier for sure.

Reply:
Yes, I love it! Thank you for reminding me about it :-)